modules/ROOT/pages/customize.adoc

```diff
@@ -1,101 +1,177 @@
+The https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] enables
+you to create a community with your desired brand identity, look and feel, and
+site structure by providing control over the community's components, themes,
+pages, and its general, theme, and SEO settings.
 
-Select *Community Builder* in the API Community Manager (ACM) Control Panel. By default, the
-Community Builder displays the *Home* page and the Pages drop-down list displays *Home*. You can select
-the Pages drop-down list to display a list of community pages and specify a different page for customization.
+In the API Community Manager (ACM) Control Panel, the *Community Builder* shows
+your site for editing. The builder is where you edit each page in your
+community, create new pages, and add and edit the components in each page.
 
-The *Community Builder* interface contains the customization icons described here.
+The *Community Builder* interface contains the elements described here.
+
+== Pages List
+
+The *Pages* list shows the pages included with ACM and the pages you have added,
+and enables you to select and edit each page. The included page *Home* is
+selected by default.
+
+The *Pages* list enables you to create new pages with the *New Page* option.
+
+A new page can be an *Object Page* or *Standard Page*. An *Object Page* is used
+only with Salesforce objects, so a custom community page starts as a
+*Standard Page*. A standard page can start from one of the preconfigured pages
+or as a *New Blank Page*.
+
+A new blank page is made by selecting the layout and then entering the new
+page's *Name* and *URL*. The URL is used to make links to the new page,
+including links in menus.
+
+The new page is displayed with placeholders for content based on the layout.
+
+A new page is filled in by adding components from the *Components* menu, such
+as a *Headline* component in the page header, or an ACM Lightning component
+such as an *API Version Carousel*.
+
+The *Pages* list shows the editable configuration properties for each page,
+which are described in https://help.salesforce.com/articleView?id=community_builder_manage_pages_overview.htm&type=5[Manage Your Community’s Pages and Their Properties in Community Builder, role=external, window=blank].
 
 == Components
 
-Select the *Components* lightning bolt icon in the top left navigation to display the component palette.
-The component palette includes Salesforce Community Cloud and ACM-specific components.
-Drag and drop the desired components from the component palette to the community page. Each
-component has its own configuration options that you can then access by selecting the component on the page.
+The *Components* lightning bolt icon displays the component palette, which
```

Comment:
```suggestion
Clicking the *Components* lightning bolt icon displays the component palette, which
```

Reply:
GLOBALLY unbold the component names; I will stop marking from here.

Reply:
Unbolded component names.